_pages/en_US/f3-(mac)-2.txt

```diff
@@ -0,0 +1,65 @@
+---
+title: "F3 (Mac)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This is an add-on section for checking your SD card for errors using F3X.
+
+Depending on the size of your SD card and the speed of your computer, this process can take up to several hours!
+
+This page is for Mac users only. If you are not on Mac, check out the [H2testw (windows)](h2testw-(windows)) or [F3 (Linux)](f3-(linux)) pages.
+
+### What You Need
+
+* The latest release of [F3](assets/f3mac.zip)
+
+### Instructions
+
+1. Unzip the F3 `.zip` file
+1. Insert your SD card into your computer
+1. Open Terminal
+1. `cd` into the f3 directory
```

Comment:
Possibly unclear for an unexperienced command line user; maybe place f3 in a specific directory and then tell the user exactly what to type